packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,261 @@
+    expect(history[5].amounts.length).toStrictEqual(1)
+    expect(history[5].amounts[0]).toStrictEqual('2.00000000@DFI')
+  })
+
+  it('last transaction should be send utxo to burn address', async () => {
+    expect(history[6].owner).toStrictEqual(burnAddress)
+    expect(history[6].type).toStrictEqual('None')
+    expect(typeof history[6].amounts).toStrictEqual('object')
+    expect(history[6].amounts.length).toStrictEqual(1)
+    expect(history[6].amounts[0]).toStrictEqual('10.00000000@DFI')
+  })
+
+  it('should listBurnHistory with filter on txtype None', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.NONE
+    })
+    expect(history.every(({ type }) => type === 'None')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on txtype UtxosToAccount', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.UTXOS_TO_ACCOUNT
+    })
+    expect(history.every(({ type }) => type === 'UtxosToAccount')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on txtype AccountToAccount', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.ACCOUNT_TO_ACCOUNT
+    })
+    expect(history.every(({ type }) => type === 'AccountToAccount')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on txtype CreateMasternode', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.CREATE_MASTERNODE
+    })
+    expect(history.every(({ type }) => type === 'CreateMasternode')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on txtype CreateToken', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.CREATE_TOKEN
+    })
+    expect(history.every(({ type }) => type === 'CreateToken')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on token DFI', async () => {
+    const history = await client.account.listBurnHistory({
+      token: 'DFI'
+    })
+    expect(history.every(({ amounts }) => amounts[0].includes('DFI'))).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on token GOLD', async () => {
+    const history = await client.account.listBurnHistory({
+      token: 'GOLD'
+    })
+    expect(history.every(({ amounts }) => amounts[0].includes('GOLD'))).toBeTruthy()
+  })
+
+  it('should listBurnHistory with maxBlockHeight 110', async () => {
+    const maxBlockHeight = 110
+    const history = await client.account.listBurnHistory({
+      maxBlockHeight
+    })
+    expect(history.every(({ blockHeight }) => blockHeight <= maxBlockHeight)).toBeTruthy()
+  })
+
+  it('should listBurnHistory with maxBlockHeight 500', async () => {
+    const maxBlockHeight = 500
+    const history = await client.account.listBurnHistory({
+      maxBlockHeight
+    })
+    expect(history.every(({ blockHeight }) => blockHeight <= maxBlockHeight)).toBeTruthy()
+  })
```

Comment:
test is not clear.. for this you need to assert the before after to compare the diff
eg:
```
historyBefore = await listBurnHistory()
assert(maxBlockHeight(historyBefore)).toStrictEqual(15)

historyAfter = await listBurnHistory({maxBlockHeight: 3})
assert(maxBlockHeight(historyAfter)).toStrictEqual(3)
```

Reply:
Done on 30ae17d, tell me if you find it clearer